eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/shared/equiz.module.js

```diff
@@ -0,0 +1,4 @@
+﻿(function(angular) {
```

Comment:
Why do we need this file?

That mode is already created in the common area - "Scripts/app/shared/equiz.module.js"